settings.gradle.kts

```diff
@@ -20,3 +24,15 @@ project(":frontend").projectDir = file("src/frontend")
 project(":backend").projectDir = file("src/backend")
 project(":cds-converter").projectDir = file("src/cds-converter")
 project(":schema-generator").projectDir = file("src/schema-generator")
+
+gradleEnterprise.buildScan {
+    termsOfServiceUrl = "https://gradle.com/terms-of-service"
+    termsOfServiceAgree = "yes"
+}
+
+// Gradle Enterprise enabled Build Cache, let's step back a bit...
+buildCache {
+    local {
+        isEnabled = false
```

Comment:
Won't it disable cache unconditionally? Maybe we should pass setting value here instead?

Reply:
Makes sense to change. However I always didn't understand the purpose of a local build cache: it will improve build times if you're jumping branches all the time, but nothing more. By default it's disabled unless someone mentions(!) build cache configuration.

Reply:
Fot example, github actions are able to save cache between builds.

Reply:
You run GH Actions on the same uncontainerised runners? Yes, then the cache is preferred.

Reply:
Just normal ones, on GH provided runners. gradle-build-action is able to save cache and restore it on next run.